sklearn/utils/estimator_checks.py

```diff
@@ -362,14 +349,10 @@ def _mark_xfail_checks(estimator, check, pytest):
     """Mark (estimator, check) pairs with xfail according to the
     _xfail_checks_ tag"""
     if isinstance(estimator, type):
-        # try to construct estimator instance, if it is unable to then
-        # return the estimator class, ignoring the tag
-        try:
-            estimator = _construct_instance(estimator),
-        except Exception:
-            return estimator, check
+        # the tag doesn't work with classes
+        return estimator, check
```

Comment:
@thomasjpfan can you confirm this is basically the same fix as in https://github.com/scikit-learn/scikit-learn/pull/16709 ?

Note that [this test](https://dev.azure.com/scikit-learn/scikit-learn/_build/results?buildId=15481&view=logs&j=13650d7c-49e8-54f3-0598-c3480d1c1e4f&t=97f2162d-a0c4-54dc-9fce-2fceef7172e3&l=325) would otherwise fail.

Reply:
It was the comma: https://github.com/scikit-learn/scikit-learn/pull/16954

Reply:
`_safe_tags` was really good at being safe.